tutorial/datastructures.po

```diff
@@ -649,18 +649,17 @@ msgid ""
 "When looping through a sequence, the position index and corresponding value "
 "can be retrieved at the same time using the :func:`enumerate` function. ::"
 msgstr ""
-"Lorsque vous itérez sur une séquence, la position et la valeur "
+"Lorsque vous faites une boucle sur une séquence, la position et la valeur "
 "correspondante peuvent être récupérées en même temps en utilisant la "
 "fonction :func:`enumerate`. ::"
 
 #: ../Doc/tutorial/datastructures.rst:589
-#, fuzzy
 msgid ""
 "To loop over two or more sequences at the same time, the entries can be "
 "paired with the :func:`zip` function. ::"
 msgstr ""
-"Pour faire des boucles sur deux séquences ou plus en même temps, les "
-"éléments peuvent être associés par la fonction :func:`zip` ::"
+"Pour faire une boucle sur deux ou plusieurs séquences en même temps, les "
+"éléments peuvent associés ensemble en utilisant la fonction :func:`zip` ::"
```

Comment:
Je pense que « associés ensemble » est un pléonasme.

Reply:
noté

Reply:
« Deux ou plusieurs » est un peu redondant aussi.

Reply:
comment traduire "two or more", du coup ?

Reply:
« Deux ou plus » comme avant ?
Bon après ne nous prenons pas trop la tête non plus.

Reply:
OK pour deux ou plus.